### DISCUSSION TITLE
how can i draw faster ?

### DISCUSSION THREAD

**Discussion by an99990** [facebookresearch/detectron2#4036] [Q&A]
Getting a prediction doenst take much time, but drawing the mask on the image for visualisation takes a long time.
How could I draw faster. The output prediction is a dictionary containing instances of detectron2 objects type.
How can i just retrieve the mask image and draw them using opencv ?
Or is there a faster way to draw mask other than the  Visualizer from detectron2.utils.visualizer?

thank you

**Comment by ExtReMLapin** [comment]
It sounds like a non-detectron problem ?
I personally only use detectron only to train the model.
